tests/test_other.py

```diff
@@ -3779,15 +3779,13 @@ def test_bad_triple(self):
     # compile a minimal program, with as few dependencies as possible, as
     # native building on CI may not always work well
     create_test_file('minimal.cpp', 'int main() { return 0; }')
-    try:
-      vs_env = shared.get_clang_native_env()
-    except Exception:
-      self.skipTest('Native clang env not found')
-    run_process([CLANG_CXX, 'minimal.cpp', '-target', 'x86_64-linux', '-c', '-emit-llvm', '-o', 'a.bc'] + clang_native.get_clang_native_args(), env=vs_env)
-    err = run_process([EMCC, 'a.bc'], stdout=PIPE, stderr=PIPE, check=False).stderr
+    run_process([CLANG_CXX, 'minimal.cpp', '-target', 'x86_64-linux', '-c', '-emit-llvm', '-o', 'a.bc'] + clang_native.get_clang_native_args(), env=clang_native.get_clang_native_env())
```

Comment:
It looks like this and the SIMD tests are the only ones that actually use the native clang.  That's surprising, given how much logic we have in clang_native for finding the SDKs and such! (am I wrong about that?). Also the SIMD tests don't even run on chromium CI because we don't have d8. Not sure about emscripten CI. Does it make sense to have a flag or something, or just find a more elegant way to skip the relevant tests if there's no native clang?